collect_app/src/main/java/org/fieldsight/naxa/FsInstanceChooserList.java

```diff
@@ -0,0 +1,48 @@
+package org.fieldsight.naxa;
+
+
+import android.database.Cursor;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.loader.content.Loader;
+
+import org.fieldsight.naxa.helpers.FSInstancesDao;
+import org.fieldsight.naxa.login.model.Site;
+import org.odk.collect.android.activities.InstanceChooserList;
+
+import static org.fieldsight.naxa.common.Constant.EXTRA_OBJECT;
+
+public class FsInstanceChooserList  extends InstanceChooserList {
+
+    private Site loadedSite;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Bundle bundle = getIntent().getExtras();
+        if (bundle != null) {
+            loadedSite = bundle.getParcelable(EXTRA_OBJECT);
+        }
+    }
+
+    @NonNull
+    @Override
+    public Loader<Cursor> onCreateLoader(int id, Bundle args) {
+        showProgressBar();
+        FSInstancesDao instancesDao = new FSInstancesDao();
```

Comment:
![Screenshot_1566187579](https://user-images.githubusercontent.com/11831955/63238618-07375e80-c267-11e9-81a6-09a0ad065de5.png)
I created a long list and it  `onCreateLoader()` only once
preventing multiple  `FSInstancesDao` from being created

also it says here https://stackoverflow.com/questions/24976944/oncreateloader-only-called-one-time
it only recreates it activity gets destroyed